core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateRangeIT.java

```diff
@@ -286,17 +287,22 @@ public void testSingleValueFieldWithStringDatesWithCustomFormat() throws Excepti
     }
 
     public void testSingleValueFieldWithDateMath() throws Exception {
+        int timeZoneOffset = randomIntBetween(-12, 12);
```

Comment:
Can these have .5 and .75 (Nepal, why you have weird time zone?  :confused:) parts?